appmetrics/validators.go

```diff
@@ -0,0 +1,27 @@
+package appmetrics
+
+var StringSchema = map[string]interface{}{
+	"type": "string",
+}
+
+var NavigationNavigateToCofxSchema = map[string]interface{}{
+	"type": "object",
+	"properties": map[string]interface{}{
+		"view_id": map[string]interface{}{
+			"type":      "string",
+			"maxLength": 16,
+		},
+		"params": map[string]interface{}{
+			"type": "object",
+			"properties": map[string]interface{}{
+				"screen": map[string]interface{}{
+					"enum": []string{"allowed-screen-name"},
```

Comment:
Should this be a list of allowed screen names for use in production?

Reply:
No, getting rid of this.

Reply:
Actually, if I remove this, the tests will fail. We need at least one element in the list to run tests.